File: works/2914.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');
// let input = fs.readFileSync('input.txt').toString().split('\n');
v = input[0].split(' ').map(x => parseInt(x)) 

numOfSongs = v[0] 
avg = v[1]-1  
ans = avg * numOfSongs
console.log(ans+1)

File: works/tmp/2914.dart
import 'dart:io';

void main()
{
    var input = stdin.readLineSync()!.split(' ');
    int numOfSongs = int.parse(input[0]);
    int avg = int.parse(input[1]) - 1;
    int ans = avg * numOfSongs;
    print(ans+1);
}